experiments/compositions/composition/cmd/main.go

```diff
@@ -131,6 +131,13 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "ExpanderVersion")
 		os.Exit(1)
 	}
+	if err = (&controller.GetterReconciler{
```

Comment:
This feels a bit off. Deal with it for now but shouldn't the instanced "expander" reconcilers be triggered by the ExpanderVersionReconciler? If thats not true then how do we dynamically add a expander/plugin at runtime?

Reply:
You are right. This is not required. Its only for CR validation. Validation can be done as part of CRD definition.